.github/workflows/downstream_tests_slow.yml

```
@@ -0,0 +1,62 @@
+name: Test Downstream Libraries - Slow
+
+on: 
+  workflow_call:
```

Can we have `workflow_dispatch` as well for a manual trigger, just in case?

```suggestion
  workflow_call:
  workflow_dispatch:
```

---

uh yes I forgot :) thanks